### DISCUSSION TITLE
MIB2 High navigation data problem

### DISCUSSION THREAD

**Discussion by brian7777dv** [Mr-MIBonk/M.I.B._More-Incredible-Bash#340] [Q&A]
Hi guys! 
**Can you tell me what am I doing wrong here?**

Audi Q2, with no stock navigation.
MHI2_ER_AU276_P5600 MU1498
I patched it with m.i.b.
All activation codes are now legal including
023000ee - legal
0004100 - legal
New maps for EU (23/24) installed without a problem.
All done with new 64gb (fat32) SD cards.

In the main menu now I have 2 new icons (maps & navigation), but when I try to enter one of them I get (blue) Navigation note - The navigation data are invalid"

**How can I fix this?**

**Comment by aledepaoligampel** [comment]
Hi @brian7777dv, can you fix your problem? I have the same issue.

**Reply by Goffer458** [reply]
Hi  @brian7777dv & @aledepaoligampel  . i'm currently with Q2 with MHI2_ER_AU276_P5510 1477 . Where do you find MHI2_ER_AU276_P5600 1498 ? i've searched on MIB solution FW but can' find.

**Reply by Goffer458** [reply]
Find it : Go To green menu / Navigation / Add Navigation (select your zone) / Enable Navigation ON. Then Unit restart on Nav is ok :)

**Reply by kabasm** [reply]
Could you share a link to the firmware MHI2_ER_AU276_P5600 1498? I also can’t find the P5600 anywhere. Although mib-helper claims that this is the latest version, only P5088 is on mibsolution.one.

**Reply by AlexRSmith95** [reply]
Has anybody got the link for this FW?

**Reply by antostorny** [reply]
Hi,

In order to fix you need to code the navigation correctly. I can help for free regarding it, but i kindly ask you to share the update P5600 :) i need it
